Wallie/pixab.py

```diff
@@ -19,8 +29,9 @@ def pixabay_parse_resp(subject):
     if subject is not None:
         resp = pix.search(q=subject, per_page=4, image_type="photo")
     else:
-        # Carry out default search. No random endpoint for pixabay API.
-        resp = pix.search(per_page=4, image_type="photo")
+        # give wider random image selections
+        resp = pix.search(q=randomize_query(), per_page=4,
+                          image_type="photo", page=randint(1, 10))
```

Comment:
`randint(1,10))`. Staring from `1` would mean that the first page (indexed at `0`) may never be chosen? Or maybe i'm incorrect here?

Reply:
Yes. As intended. Pixabay will return error if I use `page=0`. This URL for instance - https://pixabay.com/api/?key=API_KEY&q=yellow+flowers&image_type=photo&page=0

Reply:
I did a little bit test on the code too. It will return the out of valid range error if I did set it to `page=randint(0, 10)` after a few attempts.